aodndata/moorings/products_handler.py

```diff
@@ -106,6 +108,31 @@ def _get_input_files(self):
         self.input_file_collection.download(self._upload_store_runner.broker, self.temp_dir)
         # TODO: Replace temp_dir above with cache_dir?
 
+    def _get_old_product_files(self):
+        """Get a list of the currently published aggregated_timeseries files for the site being processed."""
+
+        filter_list = [PropertyIsEqualTo(propertyname='site_code', literal=self.product_site_code),
+                       PropertyIsEqualTo(propertyname='data_category', literal='aggregated_timeseries')
+                       ]
+        wfs_features = self._get_wfs_features(filter_list, propertyname=['url'])
+
+        self.old_product_files = {}
+        var_pattern = re.compile(r'FV01_([A-Z0-9-]+)-aggregated')
```

Comment:
Apparently this can be promote to a global - "AGG_VAR_PATTERN" or something.

Reply:
Yeap, but despite being only used within the private func,  it's a data condition not method. In other words, the nature of the definition can lead to re-use in the future.

Reply:
Ok. done